app/server.jsx

```diff
@@ -16,6 +16,34 @@ const clientConfig = {
 // configure baseURL for axios requests (for serverside API calls)
 axios.defaults.baseURL = `http://${clientConfig.host}:${clientConfig.port}`;
 
+
+const analtyicsScript =
+  typeof trackingID === "undefined" ? ``
+  :
+  `<script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+    ga('create', ${trackingID}, 'auto');
+    ga('send', 'pageview');
+  </script>`;
+
+
+/*
+ * To Enable Google analytics simply replace the hashes with your tracking ID
+ * and move the constant to above the analtyicsScript constant.
+ *
+ * Currently because the ID is declared beneath where is is being used, the
+ * declaration will get hoisted to the top of the file.
+ * however the assignement  does not, so it is undefined for the type check above.
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#var_hoisting
+ */
+const trackingID  = "'UA-########-#'";
+
```

Comment:
Would it make more sense to have this tracking ID in our [appConfig](https://github.com/choonkending/react-webpack-node/blob/master/server/config/appConfig.js) so that users can change that value without having to move the constant above the line?